pcpostprocess/scripts/run_herg_qc.py

```diff
@@ -107,6 +107,9 @@ def main():
     sys.modules['export_config'] = export_config
     spec.loader.exec_module(export_config)
 
+    data_list = os.listdir(args.data_directory)
+    export_config.D2S_QC = {x: y for x, y in export_config.D2S_QC.items() if
+                            any([x == '_'.join(z.split('_')[:-1]) for z in data_list])}
```

Comment:
I think this should work fine. But this could throw an error if the files aren't set out properly. Maybe we should just check that there is at least one "_" in all of these files. I think it's mostly safe to assume that the data folders will be formatted correctly, but there's nothing stopping someone from dragging any other sort of file in there!